packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2926,6 +2926,7 @@ export default defineComponent({
 				}
 
 				if (
+					// @ts-expect-error Deprecated file
```

Comment:
`!(this.workflowPermissions.update ?? this.projectPermissions.workflow.update)` will never be nullish but I'd rather not touch this, like with `no-constant-binary-expression` [last time](https://github.com/n8n-io/n8n/pull/7670/files#r1387991176). This legacy file will be deprecated soon anyway.